lib/cocoapods/installer/xcode/target_validator.rb

```diff
@@ -34,7 +36,7 @@ def initialize(aggregate_targets, pod_targets)
         def validate!
           verify_no_duplicate_framework_and_library_names
           verify_no_static_framework_transitive_dependencies
-          verify_swift_pods_swift_version
+          verify_swift_pods_swift_version if installation_options.integrate_targets?
```

Comment:
can we also add a test for what swift version should be used when integrating targets is disabled?

Reply:
hm good point......what version should it be set to?

Reply:
The highest version the pod supports?

Reply:
and what if the pod does not specify `swift_version`?

Reply:
Then we have to error, right?

Reply:
This class does not _set_ the swift version for the target, only validates it. I'd argue none of these checks should happen if integration is off.

Reply:
Wouldn't that be a reason to fail validation?

Reply:
I think so, @amorde